cmd/syncthing/gui.go

```diff
@@ -270,10 +269,12 @@ func (s *apiService) Serve() {
 	postRestMux.HandleFunc("/rest/system/restart", s.postSystemRestart)        // -
 	postRestMux.HandleFunc("/rest/system/shutdown", s.postSystemShutdown)      // -
 	postRestMux.HandleFunc("/rest/system/upgrade", s.postSystemUpgrade)        // -
-	postRestMux.HandleFunc("/rest/system/pause", s.postSystemPause)            // device
-	postRestMux.HandleFunc("/rest/system/resume", s.postSystemResume)          // device
 	postRestMux.HandleFunc("/rest/system/debug", s.postSystemDebug)            // [enable] [disable]
 
+	// Deprecated
+	postRestMux.HandleFunc("/rest/system/pause", s.deprecatedMakePauseHandler(true))   // device
```

Comment:
So just to be clear, I am fine if we want to add new calls and new events, I am just worried about the 209 rest calls to get current folder state for some people.